app/src/accountContext.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
name file AccountContext pls. this is new for use but all classes should be capital from now please. only index and files without a single class will be non-caps